external/netcoreapp/netcoreapp.depproj

```diff
@@ -28,8 +18,7 @@
   <ItemGroup>
     <!-- for all configurations this project provides refs for that configuration -->
     <BinPlaceConfiguration Include="$(Configuration)">
-      <RefPath Condition="'$(BinPlaceRef)' == 'true'">$(RefPath)</RefPath>
```

Comment:
Why make this change?

Reply:
Because now `BinPlaceRef` is always set to true. That is the way it was before I made the shims cross compile.

Reply:
We no longer binplace the runtime, we only binplace the ref that comes from the package.

Reply:
I see. Sorry I mistook the file you were changing this in. Thought it was a targets file.